common/djangoapps/edxmako/paths.py

```diff
@@ -6,9 +6,9 @@
 import contextlib
 import hashlib
 import os
-import six
 
 import pkg_resources
+import six
```

Comment:
This can be removed after updating the only use of package `six` on line 57 as `six.b(data)` --> `data.encode()`.
```suggestion
```

Reply:
Will do in next iteration.